tailwind.config.js

```diff
@@ -58,8 +58,35 @@ module.exports = {
       {
         dark: {
           ...require("daisyui/src/theming/themes")["[data-theme=dark]"],
-          primary: "#a78bfa",
-          "base-content": "#f5f3ff",
+          "base-100": "#ffffff",
+          "base-200": "#8D9DB6",
+          "base-300": "#2B5F82", // TODO:confirm in code
```

Comment:
`base-100` should be the darkest (just like in the `light` theme, it's the lightest).